app/lib/efile/az/az140_calculator.rb

```diff
@@ -141,9 +141,9 @@ def calculate_line_32
 
       def calculate_line_35
         # Subtotal after additions and subtractions
-        subtractions = line_or_zero(:AZ140_LINE_29A) + line_or_zero(:AZ140_LINE_29B)
-        (30..32).each do |line_num|
-          subtractions += line_or_zero("AZ140_LINE_#{line_num}")
+        subtraction_lines = ["28", "29A", "29B"] + (30..32).to_a
```

Comment:
i don't know if this is more misleading than the previous implementation.

Reply:
this is nice imo! 30..32 are subtraction lines too so this makes sense.